.github/workflows/ci.yml

```
@@ -71,17 +71,24 @@ jobs:
+        run: |
+          sed -i 's/DEV_MODE=False/DEV_MODE=True/' /opt/openwisp/docker-openwisp/.env
+          docker compose --file /opt/openwisp/docker-openwisp/docker-compose.yml up --detach --force-recreate
+
       # The test suite needs to create files during execution. Because the deploy
       # script runs with `sudo`, the installation directory ends up owned by root,
       # making it inaccessible to the unprivileged CI user. This step fixes the
       # ownership so the test suite can write the required files.
       - name: Fix permissions for CI user
-        if: ${{ !cancelled() && steps.auto_install_upgrade.conclusion == 'success' }}
+        if: ${{ !cancelled() && steps.enable_development_profile.conclusion == 'success' }}
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:15: Double quote to prevent globbing and word splitting

(shellcheck)